docs/Templates/SpikeOutcome-Template.md

```diff
@@ -2,53 +2,40 @@
 
 ==================
 
-**Spike:** Spike_No
+**Spike:** Splashkit_camera
```

Comment:
This supposed to be a number; but I would just remove this line since I don't think is an actual Spike?